ext/drreg/drreg.dox

```diff
@@ -42,7 +42,11 @@ instrumentation pass scheme.  The results are most efficient when used in
 the \p drmgr instrumentation insertion phase.  Use outside of the insertion
 phase, such as during \p drmgr's instrumentation-to-instrumentation
 transformation phase, or use outside of \p drmgr entirely, is supported,
-but duplicate spills and restores may not be optimized away.  When using
+but duplicate spills and restores may not be optimized away.  Also, for
+non-insertion phases, the responsibility to restore a spilled app value
+before an app read, and to re-spill the app value after an app write, lies
+with the user; in the insertion phase, this is done automatically, but it
+needs to be done manually in non-insertion phases. When using
```

Comment:
Maybe update my `XXX i#5121: multi-phase` comment in drreg_insert_restore_all() as well.

Reply:
Done. PTAL.